src/utils/featheryClient/integrationClient.ts

```diff
@@ -70,16 +74,19 @@ export default class IntegrationClient {
 
   _fetch(
     url: any,
-    options: any,
+    options?: any,
     parseResponse = true,
     propagateNetworkErrors = false
   ) {
     const { sdkKey } = initInfo();
-    const { headers, ...otherOptions } = options;
+    const { headers, ...otherOptions } = options ?? {};
```

Comment:
above this, just do `options = options ?? {};` and we dont need any of the rest of this logic

Reply:
please revert line 81, the ?? {} is not necessary anymore

Reply:
@bo-dun-1 done.